plugin/src/main/scala/Giter8Plugin.scala

```diff
@@ -135,13 +135,24 @@ object Giter8Plugin extends sbt.AutoPlugin {
         val dir     = (sourceDirectory in Test).value
         val metadir = (baseDirectory in LocalRootProject).value / "project"
         val file0   = dir / "g8" / "test"
+
+        // we should only use file0 if its an exisiting file
+        // otherwise, we fallback to test.script
+        val defaultTestScript = 
+          if (file0.exists && file0.isFile) file0
+          else dir / "g8" / "test.script"
```

Comment:
While https://github.com/sbt/sbt/issues/630 defines that a scripted test can use either "test" or "test.script", giter8 ignores "test.script" and only cares about "test", "giter8.test" or "g8.test".

Moreover, if a Play project using the default layout is built with giter8, it will always try to use the "test" folder as the script file to run. Which will of course fail. 

Basically, the change here is saying, if there is a "test" file we will use it, probably defined as such by the user, otherwise we will use "test.script". 

In case of Play templates, that will work just fine because there will be no conflict with "test.script". 

For existing users, there is a slightly different behaviour. In case the user is not defining its own script, we will create it as "test.script", instead of "test". In case the user is defining is own "test" file, then we respect its decision and keep using "test" as default.